contrib/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+# Change dir to the script's location
+cd $(pwd)
+
+# Delete the existing gre module because on some
+# systems depmod is not enough to override the
+# old gre module.
+find /usr/lib/modules/ -name "gre.ko*" -delete
+
+# If called from contrib folder, change dir up
+dirname=$(basename $(pwd))
+if [[ "$dirname" == "contrib" ]]; then
+  cd ..
+fi
+
+# Build eoip tool
+make clean
+make
+make install
+
+# Detect Kernel Major & Minor
+unamer=$(uname -r)
+kernel=(${unamer//./ })
+cd "out-of-tree-${kernel[0]}.${kernel[1]}.x"
+
+# Build eoip module
+make
+make install
+depmod
+
+# Remove the old module, continue on error
+rmmod gre || true
```

Comment:
`rmmod` will fail (e.g. in case the `pptp` module is loaded and `gre` is in use). There is no use case where only the `gre` module is loaded on its own. I think that it is better to detect the situation and inform the user, because unloading may break something else.

```
# lsmod|grep '^gre '
gre                    16384  2 eoip,pptp
```

Reply:
I will add this, but in my testing rmmod will fail. It doesn't automatically remove the dependency pptp and will just fail to remove gre.